docs/shipping/Containers/docker.md

```diff
@@ -40,6 +40,11 @@ to a lower value (for example, `20m`).
 
 * Version 0.1.0 of docker-collector-logs includes breaking changes. Please see the project's [change log](https://github.com/logzio/docker-collector-logs#change-log) for further information.
```

Comment:
The 'Upgrading to a newer version' section and the reference to the docker-collector-logs with Filebeat repository would be better placed under the Filebeat tab